lib/classroom_api.rb

```diff
@@ -143,15 +150,19 @@ def update_all_classroom_characteristics
       end
       facility_id = room.facility_code_heprod
       rmrecnbr = room.rmrecnbr
-
       result = get_classroom_characteristics(ERB::Util.url_encode(facility_id))
       if result['errorcode'] == "ERR429"
         @log.api_logger.debug "update_all_classroom_characteristics, error: API return: #{result['errorcode']} - #{result['error']} after #{number_of_api_calls_per_minutes} calls"
-        number_of_api_calls_per_minutes = 1
-        sleep(61.seconds)
-        result = get_classroom_characteristics(ERB::Util.url_encode(facility_id))
-      end
-      if result['success']
+        number_of_api_calls_per_minutes = 0
+        sleep(6.seconds)
```

Comment:
Should this be 61.seconds?

Reply:
Sorry, 6.seconds were for testing, I missed one place to change it back to 61.seconds. 
Fixed that.